librz/bin/format/mach0/mach0.c

```diff
@@ -4186,29 +4192,75 @@ void MACH0_(mach_headerfields)(RzBinFile *bf) {
 		}
 		switch (lcType) {
 		case LC_BUILD_VERSION: {
-			cb_printf("0x%08" PFMT64x "  platform    %s\n",
-				pvaddr, build_version_platform_to_string(rz_buf_read_le32_at(buf, addr)));
-			cb_printf("0x%08" PFMT64x "  minos       %d.%d.%d\n",
-				pvaddr + 4, rz_buf_read_le16_at(buf, addr + 6), rz_buf_read8_at(buf, addr + 5),
-				rz_buf_read8_at(buf, addr + 4));
-			cb_printf("0x%08" PFMT64x "  sdk         %d.%d.%d\n",
-				pvaddr + 8, rz_buf_read_le16_at(buf, addr + 10), rz_buf_read8_at(buf, addr + 9),
-				rz_buf_read8_at(buf, addr + 8));
-			ut32 ntools = rz_buf_read_le32_at(buf, addr + 12);
-			cb_printf("0x%08" PFMT64x "  ntools      %d\n",
-				pvaddr + 12, ntools);
+			ut32 platform;
```

Comment:
And here

Reply:
This code should be rewritten. I just don't have the time to do it.